src/zkevm_specs/evm/execution_result/jump.py

```diff
@@ -0,0 +1,31 @@
+from ..step import Step
+from ...util import linear_combine, le_to_int, EMPTY_CODE_HASH
+from ..opcode import Opcode
+from .execution_result import ExecutionResult
+
+
+def jump(curr: Step, next: Step, r: int, opcode: Opcode):
+    assert opcode == Opcode.JUMP
+
+    # pop `dest`
+    dest = curr.decompress(curr.stack_pop_lookup(), 32, r)
+
+    # MaxCodeSize = 24576 in evm which occupies max 14 bytes
```

Comment:
14 bytes means 112 bits. I think you mean 14 bits here, right? Then at most 2 bytes can be non-zero.

Reply:
btw 24576 should be at most 15 bits

Reply:
yeah, thanks & fixed